src/web.rs

```diff
@@ -105,6 +139,10 @@ impl WebImpl {
 pub trait SurfaceExtWeb: Sized {
     /// Creates a new instance of this struct, using the provided [`HtmlCanvasElement`].
     fn from_canvas(canvas: HtmlCanvasElement) -> Result<Self, SoftBufferError>;
+
+    /// Creates a new instance of this struct, using the provided [`HtmlCanvasElement`].
+    #[cfg(web_sys_unstable_apis)]
```

Comment:
Is this `cfg` stable? If so, the method documentation should mention it (and it should also be `doc(cfg(..))`)

Reply:
It's Rust stable, but it's not stable API in `web-sys`.
> (and it should also be `doc(cfg(..))`)

Just tried it, `feature(doc_auto_cfg)` already takes care of that.